packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -36,6 +36,15 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
     return () => clearInterval(interval);
   });
 
+  const handleUrlNavigation = (url?: string, target?: Redirect['target']) => {
+    const isRelativeUrl = url?.startsWith('/');
+    if (url && !isRelativeUrl) {
+      window.open(url, target, DEFAULT_REFERRER);
+    } else if (url && isRelativeUrl) {
+      navigate(url);
+    }
+  };
```

Comment:
handle the redirect URL navigation:
- when the URL is absolute use the `window.open`
- otherwise use: `props.routerPush` or `window.history.pushState`

Reply:
@LetItRock allow me to suggest the following:

Navigation logic should live in one place and handled by a single function. I suggest we merge this logic inside the default [navigate function](https://github.com/novuhq/novu/pull/6444/files#diff-f3efe00becd29459f65a5eed571053b33d0f06dc5fff9fab98a9d1cfa69917e6R57). 

Target should be an optional argument.

That way, the default function is smart enough to handle all cases.

Reply:
yes, you are right, I noticed that after I opened PR, will update it